src/checkstyle.rs

```diff
@@ -48,7 +48,7 @@ impl CheckstyleDoc {
                 span.line_start as u32,
             )
         } else {
-            ("<unknown>".to_owned(), 1, 1)
+            return;
```

Comment:
I prefer more conservative handling for the error count part; I'm going to add a fix and then merge this PR.

Reply:
I thought this seemed to need discuss. Thanks!